wiki/Tournaments/o!mLN/4/en.md

```diff
@@ -0,0 +1,387 @@
+  2. [technoplanet - Inscape \[Extended Mix\] (\[Crz\]Crysarlene) \[Stage 6\: Caesura\]](https://osu.ppy.sh/beatmapsets/2432029#mania/5298443)
+- Density
+  1. [DGK - Kinki to Hiiro Gensou -The Primary Plasm- (Yuiesta, Hylotl) \[Stage 3\: fumo (w/ Yuiesta)\]](https://osu.ppy.sh/beatmapsets/2432025#mania/5298430)
+  2. [Feryquitous feat. Aitsuki Nakuru - Fake (Sped Up Ver.) (ImperialTrinity) \[Stage 7\: Faker\]](https://osu.ppy.sh/beatmapsets/2432031#mania/5298445)
+- Technical
+  1. [Miyavi - What's my Name (Paturages) \[Stage 4\: Heisenberg\]](https://osu.ppy.sh/beatmapsets/2432053#mania/5298513)
+
+## Ruleset
+
+*For the latest rules, see the [forum post](https://osu.ppy.sh/community/forums/topics/2107840)*
+
+### General rules
+
+1. This tournament is a **1v1 global open rank double-elimination tournament**.
+2. Players will first go through Qualifiers, then continue to the double-elimination bracket.
+3. Staff members that met [official tournament support rules](/wiki/Tournaments/Official_support#staff) are allowed to play.
+4. Players and staff members must have read this ruleset entirely.
+5. Any rule changes or unexpected occurrences will be announced in the [Discord server](https://discord.gg/SPQtt3s).
+
+### Mappool format
+
+The maps in this tournament will only consist of long note (LN) oriented maps, and cover 4 sub-categories of LNs which are used to categorise all mappools after this stage. Details can be found below:
+
+![Mappool structure](img/mappool-structure.png "mappool structure of o!M LN4")
+
+### Registrations
+
+1. **Any** rank players from **any** region can register this tournament.
+2. You must fill [Player Signup form](https://docs.google.com/forms/u/0/d/e/1FAIpQLSfQGnO66J7b22TgVmCTjhbmc3nyCpheLAup4wlMupt_3MUwcw/) to finish register.
+3. If player passed [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) after register, they are able to play qualifiers.
+
+### Stage regulations
+
+#### Qualifiers
+
+1. After the screening phase has concluded, players eligible to play will participate in tournament play.
+2. In the first weekend, players will play 7 qualifiers maps in sequence up to 2 times, in a scheduled lobby (they may choose not to play the 2nd time if desired).
+3. The 7 qualifiers maps will cover 4 sub-categories of LNs which are used to categorise all mappools after this stage:
+   - Stage 1: Early-game Releases
+   - Stage 2: Early to Mid-game Wildcard (Short Duration)
+   - Stage 3: Pure Density
+   - Stage 4: Mid-game Technical
+   - Stage 5: Late-game Releases
+   - Stage 6: Mid to Late-game Wildcard (Long Duration)
+   - Stage 7: Speed Density
+4. This tournament will make use of the **Average Placement** method to seed players.
+5. In the qualification process, every player who signs up has to schedule for a lobby during the Qualfiers Stage, and play through all 7 qualifier maps in the mappool at least once, and up to 2 times.
```

Comment:
```suggestion
5. During the qualification process, every player who signed up has to schedule for a lobby during the Qualfiers stage, and play through all 7 qualifier maps in the mappool at least once, and up to 2 times.
```

Additionally, this point is also redundant with point 2. I'd suggest either merging both, using one, or using the other. But keeping both seems unnecessary.